src/com/twitter/intellij/pants/PantsManager.java

```diff
@@ -183,10 +181,6 @@ public void enhanceRemoteProcessing(@NotNull SimpleJavaParameters parameters) th
     );
   }
 
-  @Override
-  public void enhanceLocalProcessing(@NotNull List<URL> urls) {
-  }
-
   @Override
   public void runActivity(@NotNull Project project) {
   }
```

Comment:
Change in this file seems unrelated. We can land a separate patch for beautification.

Reply:
sounds good.